packages/openapi-v3/test/integration/controller-spec.integration.ts

```diff
@@ -41,7 +41,11 @@ describe('controller spec', () => {
       paths: {
         '/foo': {
           post: {
-            responses: {},
+            responses: {
+              '200': {
+                description: 'Return value of FooController.create',
+              },
```

Comment:
Interesting. Is this the test verifying how a response status with no `content` is handled?

> IIRC, `content` is an optional property. Please add a test to verify what happens when the OAI spec does not provide any value for `content`, and fix the implementation if needed.

It makes me wonder why this test passes, when the code processing the schema is not checking whether `.content` is set. Are we filling the missing `content` property somewhere else?

Reply:
See https://github.com/strongloop/loopback-next/pull/1655#discussion_r216884048